### PR TITLE
Modified common/utils/fakerUtil.js from undepreciated codes

### PR DESCRIPTION
**Pull request by Angra974**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description

When loading the website, we have a lot of waiting time.
In my localhost, i notice that I have to wait for some warnings informations to be processed.

Even when "warning" are not checked on the console, they are processed until the end.

I notice like 30 000 warning for the same @fakerUtil.js

Fixes https://github.com/reactplay/react-play/issues/1351

## Type of change

Modify existing code to undepreciated one :

faker.name is not in the api documentation anymore and use faker.person instead
faker.image.nature is now modified to faker.image.urlLoremFlickr

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Information on the problem can be notice on reactplay.io
Code was modified and tested ( no new warning associate ) on local host.

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.
Just reload the application page after modification was done.
The fakerUtils.js is for the moment used only on the react

# Checklist:

- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

# Screenshots or example output

Before change on reactplay.io 
Capture d’écran du 2023-10-17 20-22-45

before

Documentation about image in faker
Capture d’écran du 2023-10-17 20-18-52

After change : 

react-transitions

The warnings are gone and the play works well. 
But we notice some errors in the play.
Even if it's not the meaning of the PR, i've modified the play because it was just : 
* renaming a file and some minore code to add.

I've changed the threshold to 100 instead of 10 000. I think no need to have so much data for testing a play and how it works but with a smaller value, the loading time of the play will be better.

### CODE REVIEW

**Review by priyankarpal — APPROVED** [review]

File: src/plays/react-transitions/TransitionSearch.jsx [hunk 1]
```diff
@@ -36,8 +36,8 @@ export default function TrainsitionSearch({ users }) {
       ) : (
         <div className="user-cards">
           {filtered.length > 0 ? (
-            filtered.map((user) => (
-              <div class="card">
+            filtered.map((user, index) => (
+              <div className="card" key={index}>
```
Comment (Sachin-chaurasiya) [inline src/plays/react-transitions/TransitionSearch.jsx:40]: Same here

File: src/plays/react-transitions/PrioritySearch.jsx [hunk 2]
```diff
@@ -19,8 +19,8 @@ export default function PrioritySearch({ users }) {
       <input placeholder="Type a name" type="text" value={searchTerm} onChange={handleChange} />
       <div className="user-cards">
         {filtered.length > 0 ? (
-          filtered.map((user) => (
-            <div class="card">
+          filtered.map((user, index) => (
+            <div className="card" key={index}>
```
[the remark this answers is not in the document]
Comment (saddamhr) [reply src/plays/react-transitions/PrioritySearch.jsx:23]: I agree with you

```
import { faker, simpleFaker } from '@faker-js/faker';

const THRESHOLD = 10000;

export const users = Array.from(Array(THRESHOLD), () => {
  const uuid = simpleFaker.string.uuid();

  return {
    name: faker.person.fullName(),
    avatar: faker.image.avatar(),
    background: faker.image.urlLoremFlickr({ category: 'nature' }),
    id: uuid
  };
});
```

` <div className="card" key={id}>`

I think using `uuid()` would be more convenient

[the remark this answers is not in the document]
Comment (Sachin-chaurasiya) [reply src/plays/react-transitions/PrioritySearch.jsx:23]: Yes, we can use the id.

File: src/plays/react-transitions/TransitionSearch.jsx (hunk 1 above)
Comment (Angra974) [reply src/plays/react-transitions/TransitionSearch.jsx:40]: Hi all. Sorry for the late reply.
For this play, when you will type for a new search entry, all the results will change and the result will change each time with the input. 
You didn't remove/add an element from a list or something like that, but just implement a filter. So, it didn't impact the code to use the index or name, or other attribute.

uuid is a good choice but you need to use it only when you really need it ( more weight in the  final code ) but can be use to have a way to create id every where in the code base in the same way.

**Pull request closed without merging**